src/index.js

```diff
@@ -1,17 +1,12 @@
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] Please remove the eslint disable rule here too.

Reply:
Well done